.travis/run-tests-javascript.sh

```diff
@@ -2,4 +2,8 @@
 
 set -euo pipefail
 
-mvn -q -Dparallel=methods -DthreadCount=4 -Dtest=javascript.* test
+cd ../runtime/JavaScript
+npm install
```

Comment:
```suggestion
```

I think this is not needed since the runtime doesn't have any `production` dependencies only for `development`.

Reply:
want to give it a try? you might be right, but if you are the cost is negligible

Reply:
> but if you are the cost is negligible

yeah, fair point.